docker-hub/publish/index.md

```diff
@@ -2,28 +2,81 @@
 description: Docker Verified Publisher Program
 title: Docker Verified Publisher Program
 redirect_from:
-- /docker-store/byol/
-- /docker-store/certify-images/
-- /docker-store/certify-plugins-logging/
-- /docker-store/customer_faq/
-- /docker-store/publisher_faq/
-- /docker-store/trustchain/
-- /docker-store/publish/
-- /docker-hub/publish/publish/
-- /docker-hub/publish/customer_faq/
-- /docker-hub/publish/publisher_faq/
-- /docker-hub/publish/certify-images/
-- /docker-hub/publish/certify-plugins-logging/
-- /docker-hub/publish/trustchain/
-- /docker-hub/publish/byol/
+  - /docker-store/byol/
+  - /docker-store/certify-images/
+  - /docker-store/certify-plugins-logging/
+  - /docker-store/customer_faq/
+  - /docker-store/publisher_faq/
+  - /docker-store/trustchain/
+  - /docker-store/publish/
+  - /docker-hub/publish/publish/
+  - /docker-hub/publish/customer_faq/
+  - /docker-hub/publish/publisher_faq/
+  - /docker-hub/publish/certify-images/
+  - /docker-hub/publish/certify-plugins-logging/
+  - /docker-hub/publish/trustchain/
+  - /docker-hub/publish/byol/
 ---
 
-The Docker Verified and Publisher Program enables Independent Software Vendors (ISVs), development tools vendors, and platform providers to distribute  Dockerized content through Docker Hub.
+The Verified Publisher Program provides several benefits and features to Docker
+Hub publishers, such as:
 
-Development teams rely on Docker Hub for application components to build new applications. The Docker differentiated content programs like the [Docker Official Images](../official_images.md) and the Docker Verified Publishers provide developers with validation that these images come from trusted sources, reducing the risk of pulling malicious images from the impostor repositories.
+- Verified publisher badge
+- Insights and analytics
+- Advanced image management
+- Vulnerability scanning
+- No rate limiting
 
-The **Verified Publisher** badge included on the Docker Hub repositories indicates that these repositories are published by Docker partners that are part of the Verified Publisher program, and are qualified to be included in the developer secure supply chain. You can limit your repository search, when you use the 'Verified Publisher' filter.
+## Verified publisher badge
 
-To find out more about the Docker Verified and Publisher Program and to sign up for a partnership with Docker, visit the [Docker Verified Publisher Program](https://www.docker.com/partners/programs){:target="_blank" rel="noopener" class="_"} page.
+![Docker, Inc. org with a verified publisher badge](./images/verified-publisher-badge.png)
 
-![Verification Badge](images/VerifiedPublisherBadge.png)
+The verified publisher badge signals high quality, and trust, to developers.
+Images with this badge is verified to be of high quality, and that the content
```

Comment:
**question:**
> Images with this badge is verified...

Not a native speaker so just wanted to confirm this sentence is correct 🤔 
Shouldn't it be "Images with this badge **are** verified..."?

Reply:
Totally right! Missed that!